db/migrate/20180123173648_add_index_to_comments.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexToComments < ActiveRecord::Migration[5.1]
+  def change
+    add_index :comments, :commentable_type
```

Comment:
add_index :comments, [:commentable_id, :commentable_type]

tjhis is correct index should be

Reply:
refer to https://github.com/Ruby-IF081/heroes_team_app/pull/168/files#r163285133